lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -153,37 +153,47 @@ describe('modules/manager/npm/extract/index', () => {
       expect(res).toMatchSnapshot({ yarnLock: 'yarn.lock' });
     });
 
-    it('finds and filters .npmrc', async () => {
```

Comment:
can you please revert the test order, so the test changes are easier to review

Reply:
Certainly